src/App.js

```diff
@@ -214,19 +214,24 @@ function App() {
     }
 
     // Update existing user
-    const updateUser = async (body) => {
+    const updateUser = async (updateUserInfo) => {
         const res = await fetch(
             `https://bring-me-home-backend.herokuapp.com/users/`,
             {
                 method: 'PUT',
                 headers: { 'Content-type': 'application/json',
                 'Authorization': cookies.user.token },
-                body: JSON.stringify(body)
+                body: JSON.stringify(updateUserInfo)
             }
         )
 
-        if (res.ok) {
-            //console.log("User updated")
+        if (res.ok) {   
+            const data = await res.json()
+            removeCookie('user')
```

Comment:
You have to set the path for which you want the cookie to be removed. You can refer to the logout function.